IoTuring/Entity/Deployments/Wifi/Wifi.py

```diff
@@ -0,0 +1,433 @@
+import re
+from socket import AddressFamily
+
+import psutil
+import locale
+
+from IoTuring.Configurator.MenuPreset import MenuPreset
+from IoTuring.Entity.Entity import Entity
+from IoTuring.Entity.EntityData import EntitySensor
+from IoTuring.Entity.ValueFormat import ValueFormatterOptions
+from IoTuring.MyApp.SystemConsts import OperatingSystemDetection as OsD
+
+VALUEFORMATTEROPTIONS_DBM = ValueFormatterOptions(ValueFormatterOptions.TYPE_RADIOPOWER)
+VALUEFORMATTEROPTIONS_PERCENTAGE = ValueFormatterOptions(
+    ValueFormatterOptions.TYPE_PERCENTAGE
+)
+
+WIFI_CHOICE_STRING = "Name: {:<15}, IP: {:<16}, MAC: {:<11}"
+
+CONFIG_KEY_WIFI = "wifi"
+
+SIGNAL_UNIT = "dBm"  # windows also supports "%"
+SHOW_NA = False  # don't show not available extraAttributes
+
+KEY_SIGNAL_STRENGTH_PERCENT = "signal_strength_percent"
+KEY_SIGNAL_STRENGTH_DBM = "signal_strength_dbm"
+
+# WINDOWS
```

Comment:
Why  do we need so much KEYS? I think the most important 5-10 would be enough (state, ssid, ip and mac address, signal level, etc.)

Take a look at the HomeAssistant mobile app, there are only 10 wifi related sensors there.

Reply:
I reduced them to be less than 10 for each OS.
Though i just deleted some that i valued the less.